frontend-react/src/network/api/Organizations/SettingRevisions.ts

```diff
@@ -0,0 +1,34 @@
+import { API } from "../NewApi";
```

Comment:
@doug-s-nava This was my start at doing the API the new way. But you'll see I added `OrgSettingRevisionsResorce.ts` to get the rest of the code working first.

Reply:
ahah! NewApi is one of the old ways. Take a look at https://github.com/CDCgov/prime-reportstream/blob/44b7a1e0d21fa8e905e814887d4b249326e18ef5/frontend-react/src/pages/admin/value-set-editor/ValueSetsDetail.tsx#L214 for an example of the new way, or docs here https://github.com/CDCgov/prime-reportstream/blob/master/frontend-react/docs/data-fetching-patterns.md. Happy to pair whenever there's a good moment